SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -118,53 +120,53 @@ class PaymentQueueController: NSObject, SKPaymentTransactionObserver {
         let message = "SwiftyStoreKit.completeTransactions() must be called when the app launches."
         assert(completeTransactionsController.completeTransactions != nil, message)
     }
-
+    
     func startPayment(_ payment: Payment) {
         assertCompleteTransactionsWasCalled()
         
         let skPayment = SKMutablePayment(product: payment.product)
         skPayment.applicationUsername = payment.applicationUsername
         skPayment.quantity = payment.quantity
         
-        if #available(iOS 12.2, tvOS 12.2, OSX 10.14.4, *) {
+        if #available(iOS 12.2, tvOS 12.2, OSX 10.14.4, watchOS 6.2, *) {
             if let discount = payment.paymentDiscount?.discount as? SKPaymentDiscount {
                 skPayment.paymentDiscount = discount
             }
         }
         
-#if os(iOS) || os(tvOS)
-        if #available(iOS 8.3, *) {
+        #if os(iOS) || os(tvOS) || os(watchOS)
```

Comment:
Should this also have an exclusion for watchOS below Swift 5.3?

Reply:
no, is was missing before that always worked priori 5.3